### PR TITLE
fast fixes for fxa styles

### PR DESCRIPTION
**Pull request by johngruen**
This PR provides a bunch of papercut fixes to CSS assocated with accounts feature including:

* Hiding disabled buttons on shot page
* fixing layout nits on settings page when user has avatar of username specified in FxA
* fixes annotation feature so that preview image is at same height
* fixes spacing and layout of upsell ui
* adds a bit of depth to the favorite icon on the shot index page
* fixes the positioning of the shot title editor
* flips the order of the save cancel buttons in editor view to match other button layouts
* Improves spacing of shot subtitles on shot page

### CODE REVIEW

File: static/css/partials/_header.scss [hunk 1]
```diff
@@ -155,6 +146,7 @@
   color: $black;
   background: #efeff1;
   width: 100%;
+  lin-ehgiht: 1.6rem;
```
Comment (punamdahiya) [inline static/css/partials/_header.scss]: typo should be line-height

File: static/css/frame.scss [hunk 2]
```diff
@@ -535,8 +533,9 @@ body {
   box-shadow: $large-box-shadow;
   display: flex;
   justify-content: center;
-  margin: 20px auto;
   z-index: 9999;
+  position: relative;
```
Comment (punamdahiya) [inline static/css/frame.scss]: Adding position relative here is hiding top of really long shots behind editor-header, making that area inaccessible for editing (highlight, crop and pen), recommend to remove canvas relative positioning

Comment (johngruen) [reply static/css/frame.scss]: We can resolve this by attaching `pointer-events: none` to the `.editor-header` class

Comment (johngruen) [reply static/css/frame.scss]: and attaching pointer events to the little controls

File: static/css/partials/_header.scss [hunk 3]
```diff
@@ -9,12 +9,10 @@
 
   @include respond-to("large") {
     @include flex-container(row, center, center);
-    flex-basis: $grid-unit * 4.8;
   }
 
   background-color: $light-default;
   display: flex;
-  flex-basis: $grid-unit * 6.5;
   height: 100%;
```
[the remark this answers is not in the document]
Comment (johngruen) [reply static/css/partials/_header.scss]: I think we can just remove the height: 100% here instead!

**Comment by johngruen** [comment]
@punamdahiya i think i've got everything looking really good with the editor now :)

File: static/css/partials/_header.scss [hunk 4]
```diff
@@ -60,12 +57,6 @@
     position: relative;
   }
 
-  a.nav-button {
-    &:focus {
-      outline: 1px dotted $black;
-    }
-  }
-
```
Comment (punamdahiya) [inline static/css/partials/_header.scss]: nit: Removing this shows a blue highlight when settings and myshots button are pressed in chrome

**Comment by punamdahiya** [comment]
@johngruen Editor looks so much better!! a minor nit noted around settings button when pressed in chrome. Rebasing PR should also pass tests now. Thanks for working on this!

**Comment by SoftVision-PaulOiegas** [comment]
> Hiding disabled buttons on shot page

Wouldn't be better to stimulate the user to Sign up / Sign In through a message when he clicks / hovers the "Fav" button instead removing it while not logged? You can also mention the benefits of Fav somehow.

It's a big change from the previous design and users might see the removal of the option to set the expiration time as a missing functionality.

**Pull request merged**